scripts/earthengine/process_events.py

```diff
@@ -159,10 +162,19 @@
 
     # Output settings.
     'event_type': 'FloodEvent',
+    'output_delimiter': ',',
     'resolve_affected_place_latlng': True,
     'output_affected_place_polygon': 'geoJsonCoordinatesDP1',
     'polygon_simplification_factor': 0.1,
 
+    # Output svobs per place
+    'output_place_svobs': True,
```

Comment:
For my understanding, where are the keys described?

Also, wonder if we can use separate files for config constants? (later PR is fine)

Reply:
Moved the config to event_config.py with more comments describing each setting.
That is the doc for config for now :)